Lib/traceback.py

```diff
@@ -684,8 +686,8 @@ def format(self, *, chain=True):
             if indent == 0:
                 yield from te.format(chain=chain)
             else:
-                indent_str = ' '*indent
-                yield indent_str + '-'*(60-indent) +'\n'
+                indent_str = ' '*self.indent_size*indent
+                yield indent_str + '-'*(60-len(indent_str)) +'\n'
```

Comment:
What if indent_str is longer than 60? I wonder if we should just always show exactly 60 dashes.